### DISCUSSION TITLE
Home menu search

### DISCUSSION THREAD

**Discussion by MrNobodyX3** [ChristopherKlay/StadiaEnhanced#4] [Ideas & Requests]
The library of games is big enough to need a search bar however, it would be nice to get that same treatment to the home menu. You personal collection.

**Comment by ChristopherKlay** [comment]
This is actually already in the source right now, just not enabled in `v1.3.0` due to a couple of issues, like Stadia selecting games when you press backspace. Those will be fixed and it will be released when `v1.4.0` is ready (:

**Comment by ChristopherKlay** [comment]
So, turns out this might actually not be possible the way i wanted it to work.

Google basically has a "listener" running on the entire page that controls key-inputs on the home screen (but not the store page for some reason, which is why you can't actually navigate with the arrow keys there..) for UI control. While it's extremely easy to get a search bar up there that filters games, you.. can't actually use backspace, the enter key, or even the arrow keys to select a different letter, because Googles event listener straight up catches the input (i.e. if you try to delete a letter with backspace, you just select the next option or the first game in your library). Since a search bar without a way to correct spelling mistakes or even delete a single letter would be insanely tiring to use, i will exclude this from the release version for the time being.

I will keep the issue up here and continue looking for a smooth solution, it just won't be included in the next possible build.
